lib/samson/form_builder.rb

```diff
@@ -1,18 +1,31 @@
 # frozen_string_literal: true
 module Samson
   class FormBuilder < ActionView::Helpers::FormBuilder
+    SPACER = " ".html_safe
+
     def input(attribute, as: :text_field, help: false, label: false, input_html: nil, &block)
       raise ArgumentError if block && input_html
+      input_html ||= {}
+      label ||= attribute.to_s.humanize
+      help = (help ? @template.additional_info(help) : "".html_safe)
 
       block ||= -> do
-        public_send(as, attribute, {class: "form-control"}.merge(input_html || {}))
+        public_send(as, attribute, input_html)
       end
 
       content_tag :div, class: 'form-group' do
-        content = label(attribute, label, class: "col-lg-2 control-label")
-        content << content_tag(:div, class: 'col-lg-4', &block)
-        content << @template.additional_info(help) if help
-        content
+        if as == :check_box
+          content_tag(:div, class: "col-sm-offset-2 col-sm-10 checkbox") do
+            label(attribute) do
+              block.call.dup << SPACER << label << SPACER << help
```

Comment:
why is it necessary to `dup`? If label and help are empty the spacers are being included for no reason... this is getting more complicated than I expected.